### DISCUSSION TITLE
Having a color picker in IDE while setting css variables for Tailwind

### DISCUSSION THREAD

**Discussion by maxuapro** [tailwindlabs/tailwindcss#14570] [Help] [answered]
Hi, I stumbled upon this problem: I'm trying to set up root and dark theme in global css file for Tailwind, and it seems I can only write a color code in a following format: --color-something: 50 50 50; and then use in in Tailwind config like:
`something: "var(--color-something)"` and it's ok, but:
The problem is I do not have a color picker when I put it this way: --color-something: 50 50 50 !
I have tried using --color-something: rgb(50, 50, 50) and does give the color picker, but this format just doesn't work for Tailwind config
![image](https://github.com/user-attachments/assets/53e0c550-c228-4484-b6f9-57fc0edec0a6)
![image](https://github.com/user-attachments/assets/8cd49a03-0cae-4440-8190-f3a04fa228a9)

You may ask me why don't you use just HEX colors. like --color-something: #505050?
I tried, but in this case I can't add a specific opacity like bg-opacity-50 or border-opacity-30, let alone using alpha values for the opacity..
The actual problem I'm trying to solve here is that I just want to have a color in my css variables code so I could easily set up and adjust my white and dark theme..

**Comment by wongjn** [comment, marked as answer]
You could consider using [`color-mix()`](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/color-mix) in your Tailwind config: https://play.tailwindcss.com/l3ANlxpez4?file=config

**Reply by maxuapro** [reply]
wow, thanks bro! I had no idea about that color-mix() function..
Now I can easily set my colors for different themes and adjust them. Thanks a lot
